xml/System/Activator.xml

```diff
@@ -307,6 +307,8 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
+ The method returns <see langword="null" /> for the <xref:System.Nullable%601> instances.
```

Comment:
It seems we can't use `see langword="" />` in the remarks because it's a CDATA section.
```suggestion
 The method returns `null` for the <xref:System.Nullable%601> instances.
```